.vsts-ci.yml

```
@@ -142,6 +142,12 @@ stages:
               _PublishType: none
               _SignType: test
 
+    - template: /eng/common/templates/job/source-build.yml
```

The indentation appears off here.  This may break the parsing.  The source-build leg should run for both PRs and internal builds so it should not be in the PR condition.
```suggestion
  - template: /eng/common/templates/job/source-build.yml
```

---

It was actually the parameter indentation that was off :) I've applied all your suggestions, thanks!

---

My concern is that it looks like this job is now only running for PR validation.  It needs to run in CI builds as well.  It should be moved outside of this condition - https://github.com/dotnet/sdk/blob/80fdaf4bb44ca33d380a452a88b1aa3ec395ca48/.vsts-ci.yml#L63